src/clj/game/cards/events.clj

```diff
@@ -220,7 +220,8 @@
     :msg (msg "increase the rez cost of the first unrezzed piece of ice approached by " target " [Credits]")
     :effect (effect
               (continue-ability
-                (let [bribery-x target]
+               (let [bribery-x target
+                     eid (assoc eid :x-cost true)]
```

Comment:
I don't know how we want to do it but this should be programmatic instead of individually placed in each necessary card.

Reply:
^^ I agree. There isn't currently any information on the card being x-cost as far as I can tell (it's listed as 0 - I think it's stripped out in netrunner-data). What do you suggest?

Reply:
Damn, is it really? That's a shame cuz it makes this harder. Maybe we use this fix for now and spend some hammock time thinking about a good abstraction for this.